### PR TITLE
[FEAT] Add support for DEPOSIT and WITHDRAW token transfer event in older contracts

### LINKED ISSUE #7652: Token Transfer not Indexed for older contracts
### Steps to reproduce

Contract like [this](https://explorer.fuse.io/address/0x0BE9e53fd7EDaC9F859882AfdDa116645287C629) emit DEPOSIT and WITHDRAW events instead of traditional TRANSFER events, which Blockscout does not index.

### Expected behaviour

Token Transfer should happen, and balances should get updated.

### Actual behaviour

Transaction completes but no token transfer happen, and balances do not get updated

### PR DESCRIPTION
**Pull request by MayankMittal1**
Resolves #7652 

### Enhancements
Add support for DEPOSIT and WITHDRAW token transfer event in older contracts

## Checklist for your Pull Request (PR)

  - [x] I added an entry to `CHANGELOG.md` with this PR
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from silently reappearing again.
  - [ ] I checked whether I should update the docs and did so by submitting a PR to https://github.com/blockscout/docs
  - [ ] If I added/changed/removed ENV var, I submitted a PR to https://github.com/blockscout/docs to update the list of env vars at https://github.com/blockscout/docs/blob/master/for-developers/information-and-settings/env-variables.md and I updated the version to `master` in the Version column. Changes will be reflected in this table: https://docs.blockscout.com/for-developers/information-and-settings/env-variables. 
  - [ ] If I add new indices into DB, I checked, that they are not redundant with PGHero or other tools

### CODE REVIEW

File: apps/explorer/lib/explorer/chain/token_transfer.ex [hunk 1]
```diff
@@ -74,6 +74,8 @@ defmodule Explorer.Chain.TokenTransfer do
   @typep api? :: {:api?, true | false}
 
   @constant "0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef"
+  @constant_old "0xe1fffcc4923d04b559f4d29a8bfc6cda04eb5b0d3c460751c2402c5c5cc9109c"
```
Comment (nikitosing) [inline apps/explorer/lib/explorer/chain/token_transfer.ex:77]: let's rename all `constant_old` to `weth_deposit_signature` and all `constant_old_withdraw` to `weth_withdrawal_signature`

File: apps/indexer/lib/indexer/transform/token_transfers.ex [hunk 2]
```diff
@@ -23,6 +23,14 @@ defmodule Indexer.Transform.TokenTransfers do
       |> Enum.filter(&(&1.first_topic == unquote(TokenTransfer.constant())))
       |> Enum.reduce(initial_acc, &do_parse/2)
 
+    erc20_and_erc721_token_transfers_old =
```
Comment (nikitosing) [inline apps/indexer/lib/indexer/transform/token_transfers.ex:26]: I suggest rename it to weth_transfers

**Comment by nikitosing** [comment]
@MayankMittal1 Hello, am I right that this fix already deployed on the FUSE explorer?

**Comment by MayankMittal1** [comment]
@nikitosing Yes you are right, it is already live on FUSE explorer

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,7 +3,7 @@
 ## Current
 
 ### Features
-
```
Comment (nikitosing) [inline CHANGELOG.md:6]: Please return back empty line here

**Review by nikitosing — APPROVED** [review]

**Review by sl1depengwyn — COMMENTED** [review]
wdyt about updating this test?https://github.com/blockscout/blockscout/blob/7b0703f633ff98cd5c20b6a90930b495618ea6ba/apps/indexer/test/indexer/transform/token_transfers_test.exs#L9
like this 
```elixir
test "parse/1 parses logs for tokens and token transfers" do
      [log_1, _log_2, log_3, weth_deposit_log, weth_withdrawal_log] =
        logs = [
          %{
            address_hash: "0xf2eec76e45b328df99a34fa696320a262cb92154",
            block_number: 3_530_917,
            block_hash: "0x79594150677f083756a37eee7b97ed99ab071f502104332cb3835bac345711ca",
            data: "0x000000000000000000000000000000000000000000000000ebec21ee1da40000",
            first_topic: "0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef",
            fourth_topic: nil,
            index: 8,
            second_topic: "0x000000000000000000000000556813d9cc20acfe8388af029a679d34a63388db",
            third_topic: "0x00000000000000000000000092148dd870fa1b7c4700f2bd7f44238821c26f73",
            transaction_hash: "0x43dfd761974e8c3351d285ab65bee311454eb45b149a015fe7804a33252f19e5",
            type: "mined"
          },
          %{
            address_hash: "0x6ea5ec9cb832e60b6b1654f5826e9be638f276a5",
            block_hash: "0x79594150677f083756a37eee7b97ed99ab071f502104332cb3835bac345711ca",
            block_number: 3_586_935,
            data: "0x",
            first_topic: "0x55e10366a5f552746106978b694d7ef3bbddec06bd5f9b9d15ad46f475c653ef",
            fourth_topic: "0xc89efdaa54c0f20c7adf612882df0950f5a951637e0307cdcb4c672f298b8bc6",
            index: 0,
            second_topic: "0x00000000000000000000000063b0595bb7a0b7edd0549c9557a0c8aee6da667b",
            third_topic: "0x000000000000000000000000f3089e15d0c23c181d7f98b0878b560bfe193a1d",
            transaction_hash: "0x8425a9b81a9bd1c64861110c1a453b84719cb0361d6fa0db68abf7611b9a890e",
            type: "mined"
          },
          %{
            address_hash: "0x91932e8c6776fb2b04abb71874a7988747728bb2",
            block_number: 3_664_064,
            block_hash: "0x79594150677f083756a37eee7b97ed99ab071f502104332cb3835bac345711ca",
            data: "0x000000000000000000000000000000000000000000000000ebec21ee1da40000",
            first_topic: "0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef",
            fourth_topic: "0x00000000000000000000000000000000000000000000000000000000000000b7",
            index: 1,
            second_topic: "0x0000000000000000000000009851ba177554eb07271ac230a137551e6dd0aa84",
            third_topic: "0x000000000000000000000000dccb72afee70e60b0c1226288fe86c01b953e8ac",
            transaction_hash: "0x4011d9a930a3da620321589a54dc0ca3b88216b4886c7a7c3aaad1fb17702d35",
            type: "mined"
          },
          %{
            address_hash: "0x0BE9e53fd7EDaC9F859882AfdDa116645287C629",
            block_number: 23_704_638,
            block_hash: "0x8f61c99b0dd1196714ffda5bf979a282e6a62fdd3cff25c291284e6b57de2106",
            data: "0x00000000000000000000000000000000000000000000002be19edfcf6b480000",
            first_topic: "0xe1fffcc4923d04b559f4d29a8bfc6cda04eb5b0d3c460751c2402c5c5cc9109c",
            second_topic: "0x000000000000000000000000fb76e9e7d88e308ab530330ed90e84a952570319",
            third_topic: nil,
            fourth_topic: nil,
            index: 1,
            transaction_hash: "0x185889bc91372106ecf114a4e23f4ee615e131ae3e698078bd5d2ed7e3f55a49",
            type: "mined"
          },
          %{
            address_hash: "0x0BE9e53fd7EDaC9F859882AfdDa116645287C629",
            block_number: 23_704_608,
            block_hash: "0x5a5e69984f78d65fc6d92e18058d21a9b114f1d56d06ca7aa017b3d87bf0491a",
            data: "0x00000000000000000000000000000000000000000000000000e1315e1ebd28e8",
            first_topic: "0x7fcf532c15f0a6db0bd6d0e038bea71d30d808c7d98cb3bf7268a95bf5081b65",
            second_topic: "0x000000000000000000000000e3f85aad0c8dd7337427b9df5d0fb741d65eeeb5",
            third_topic: nil,
            fourth_topic: nil,
            index: 1,
            transaction_hash: "0x07510dbfddbac9064f7d607c2d9a14aa26fa19cdfcd578c0b585ff2395df543f",
            type: "mined"
          }
        ]

      expected = %{
        tokens: [
          %{
            contract_address_hash: log_3.address_hash,
            type: "ERC-721"
          },
          %{
            contract_address_hash: log_1.address_hash,
            type: "ERC-20"
          },
          %{
            contract_address_hash: weth_withdrawal_log.address_hash,
            type: "ERC-20"
          }
        ],
        token_transfers: [
          %{
            block_number: log_3.block_number,
            log_index: log_3.index,
            from_address_hash: truncated_hash(log_3.second_topic),
            to_address_hash: truncated_hash(log_3.third_topic),
            token_contract_address_hash: log_3.address_hash,
            token_ids: [183],
            transaction_hash: log_3.transaction_hash,
            token_type: "ERC-721",
            block_hash: log_3.block_hash
          },
          %{
            token_ids: nil,
            amount: Decimal.new(17_000_000_000_000_000_000),
            block_number: log_1.block_number,
            log_index: log_1.index,
            from_address_hash: truncated_hash(log_1.second_topic),
            to_address_hash: truncated_hash(log_1.third_topic),
            token_contract_address_hash: log_1.address_hash,
            transaction_hash: log_1.transaction_hash,
            token_type: "ERC-20",
            block_hash: log_1.block_hash
          },
          %{
            amount: Decimal.new("63386150072297704"),
            block_hash: weth_withdrawal_log.block_hash,
            block_number: weth_withdrawal_log.block_number,
            from_address_hash: truncated_hash(weth_withdrawal_log.second_topic),
            log_index: 1,
            to_address_hash: "0x0000000000000000000000000000000000000000",
            token_contract_address_hash: weth_withdrawal_log.address_hash,
            token_ids: nil,
            token_type: "ERC-20",
            transaction_hash: weth_withdrawal_log.transaction_hash
          },
          %{
            amount: Decimal.new("809467672956315893760"),
            block_hash: weth_deposit_log.block_hash,
            block_number: weth_deposit_log.block_number,
            from_address_hash: "0x0000000000000000000000000000000000000000",
            log_index: 1,
            to_address_hash: truncated_hash(weth_deposit_log.second_topic),
            token_contract_address_hash: weth_deposit_log.address_hash,
            token_ids: nil,
            token_type: "ERC-20",
            transaction_hash: weth_deposit_log.transaction_hash
          }
        ]
      }

      assert TokenTransfers.parse(logs) == expected
    end

```

**Comment by MayankMittal1** [comment]
@sl1depengwyn i did the changes squash and pushed, but the pr is not getting updated

**Review by sl1depengwyn — APPROVED** [review]

**Pull request merged**